lib/sidekiq/throttled/web/summary_fix.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Throttled
+    module Web
+      module SummaryFix
+        JAVASCRIPT = <<-JAVASCRIPT
+        <script>
+          $(function ($el) {
+            var $el = $(".summary li.enqueued > a"),
```

Comment:
Thanks for the clarification. I like the simpler jquery css path a lot more as the lesser of two evils.